pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -667,86 +836,92 @@ def _distribution_new_customers(
                 s=s,
                 beta=beta,
                 T=T,
+                dims=["customer_id", "obs_var"],
             )
 
             return pm.sample_posterior_predictive(
                 self.fit_result,
                 var_names=var_names,
                 random_seed=random_seed,
-            ).posterior_predictive
+                predictions=True,
+            ).predictions
 
-    def distribution_new_customer_dropout(
+    def distribution_customer_population(
         self,
+        T: Union[None, np.ndarray, pd.Series] = None,
         random_seed: Optional[RandomState] = None,
     ) -> xarray.Dataset:
-        """Sample from the Gamma distribution representing dropout times for new customers.
+        """Pareto/NBD process representing purchases across the customer population.
 
-        This is the duration of time a new customer is active before churning, or dropping out.
+        This is the distribution of purchase frequencies given 'T' observation periods for each customer.
```

Comment:
If this function could support covariates, we can recreate the frequency PPC plot you're familiar with for covariate model evaluation with covariates. It's already conditional on a `T` array of observations.